### PR TITLE
Project assignment

### PR DESCRIPTION
**Pull request by Pav0l**

### CODE REVIEW

File: src/10_functions.py [hunk 1]
```diff
@@ -2,11 +2,19 @@
 
 # YOUR CODE HERE
 
+def is_even(num):
+  if num % 2 == 0:
```
Comment (Velsu) [inline src/10_functions.py:6]: I believe there is no need to explicitly return True or False, You can just `return num % 2 == 0` which will return proper result with shorter syntax.

File: src/12_scopes.py [hunk 2]
```diff
@@ -19,6 +20,7 @@ def outer():
     y = 120
 
     def inner():
+        nonlocal y
```
Comment (Velsu) [inline src/12_scopes.py:23]: Nice usage of `nonlocal`

**Review by Velsu — COMMENTED** [review]
Pavol great job on the assignment. You have properly completed all the exercises and achieved the MVP. Your code is clean and concise and it seems You did not have much troubles with going to Python from Javascript.

Git workflow as always flawless, great job 👍

File: src/10_functions.py (hunk 1 above)
Comment (Pav0l) [reply src/10_functions.py:6]: Thanks for pointing that out 👍

**Pull request merged**